File: module7/Collaboration/Python/GetRoomSipUri.py
import requests

url = "https://api.ciscospark.com/v1/rooms/<room ID>"

querystring = {"showSipAddress":"true"}

payload = "{\n  \"showSipAddress\": \"true\"\n}"
headers = {
    'authorization': "Bearer <your token>",
    'cache-control': "no-cache",
    'postman-token': "8386370d-5a45-f517-b347-10577b58f4f6"
    }

response = requests.request("GET", url, data=payload, headers=headers, params=querystring)

print(response.text)

File: module7/Collaboration/NodeJS/GetRoomSipUri.js
var request = require("request");

var options = { method: 'GET',
  url: 'https://api.ciscospark.com/v1/rooms/Y2lzY29zcGFyazovL3VzL1JPT00vMzRjNTExZTAtNWIzYS0xMWU2LTlmNTgtYzcwMGY5YjM0YWFl',
  qs: { showSipAddress: 'true' },
  headers: 
   { 'postman-token': '1379109c-869c-6260-63aa-7ba614e9250e',
     'cache-control': 'no-cache',
     authorization: 'Bearer NTIyN2U5NGQtY2JjZi00NGYxLWE0OTMtYzMwYTY3OWQzMDljOGE0YzA3ZTctNjY1' },
  body: '{\n  "showSipAddress": "true"\n}' };

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});
